libc/utils/docgen/sys/time.yaml

```diff
@@ -0,0 +1,9 @@
+functions:
+  select:
+    in-latest-posix: ''
+  utimes:
+    in-latest-posix: ''
+
+macros:
+  FD_SETSIZE:
+    in-latest-posix: ''
```

Comment:
> The <sys/time.h> header shall define the following as described in <sys/select.h>: FD_CLR() FD_ISSET() FD_SET() FD_ZERO() FD_SETSIZE

I take this to mean that including sys/time includes sys/select indirectly.  Therefore, sys/time.h itself isn't providing the definition of FD_SETSIZE; sys/select.h is.

So FD_SETSIZE can be removed here from sys/time.h.